packages/lib/src/vis/heatmap/HeatmapMesh.tsx

```diff
@@ -142,31 +141,34 @@ const SCALE_SHADER: Record<ScaleType, ScaleShader> = {
 
 function HeatmapMesh(props: Props) {
   const {
-    rows,
-    cols,
     values,
     domain,
-    colorMap,
     scaleType,
+    colorMap,
     invertColorMap = false,
+    textureType = TEXTURE_TYPE_BY_DTYPE[values.dtype],
```

Comment:
Would it be possible to have a `dtype` that is not a key of `TEXTURE_TYPE_BY_DTYPE` ? Making `textureType` `undefined` ?

If yes, we should account for it

Reply:
It's the case for `float64` and "big ints".

Reply:
This line is safe, since I'm restricting the type of `values` to `NdArray<CompatibleTypedArray>`.